src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java

```diff
@@ -33,7 +35,8 @@
     private final String SIX_POINT_ONE_CHANNEL = "7";
```

Comment:
is the name of the constant correct? it say six point.. and the value is 7, same with the next constant.

Reply:
ok

Reply:
Asking the creator of the class, he recommended that I not change it

Reply:
ok